### PR TITLE
Remove references to setting `cloud_storage_access_key` and `cloud_storage_secret_key` in Kubernetes

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2757
Review deadline: 10 October

## Page previews

- https://deploy-preview-799--redpanda-docs-preview.netlify.app/current/manage/kubernetes/tiered-storage/k-tiered-storage/
- https://deploy-preview-799--redpanda-docs-preview.netlify.app/current/manage/kubernetes/tiered-storage/k-remote-read-replicas/

## Checks

- [ ] New feature
- [ ] Content gap
- [x] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/partials/tiered-storage.adoc [hunk 1]
```diff
@@ -348,12 +343,32 @@ Replace the following placeholders:
...
+===== Use access keys
+
 To configure access to Google Cloud Storage with access keys instead of an IAM role:
 
 . Choose a uniform access control when you create the bucket.
 . Use a Google-managed encryption key.
 . Set a https://cloud.google.com/storage/docs/migrating#defaultproj[default project^].
-. Create a service user with https://cloud.google.com/storage/docs/authentication/managing-hmackeys[HMAC keys^] and copy the access key and secret key for the `cloud_storage_access_key` and `cloud_storage_secret_key` properties.
+. Create a service user with https://cloud.google.com/storage/docs/authentication/managing-hmackeys[HMAC keys^] and make a note of the access key and secret key.
+. Make a note of the access key and secret key.
```
Comment (Feediver1) [inline modules/manage/partials/tiered-storage.adoc]: Delete? You just said this in the previous step.

**Review by Feediver1 — APPROVED** [review]
one update suggested

File: modules/manage/partials/remote-read-replicas.adoc [hunk 2]
```diff
@@ -3,6 +3,8 @@
 include::shared:partial$enterprise-license.adoc[]
 ====
 
+:env-kubernetes: true
```
Comment (JakeSCahill) [inline modules/manage/partials/remote-read-replicas.adoc]:
```suggestion
```

File: modules/manage/partials/tiered-storage.adoc (hunk 1 above)
Comment (JakeSCahill) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
```

**Pull request merged**